test/benchmarks/benchmark_crc32.cc

```diff
@@ -56,7 +56,7 @@ class crc32: public benchmark::Fixture {
     } \
     BENCHMARK_REGISTER_F(crc32, name)->Arg(1)->Arg(8)->Arg(12)->Arg(16)->Arg(32)->Arg(64)->Arg(512)->Arg(4<<10)->Arg(32<<10)->Arg(256<<10)->Arg(4096<<10);
 
-BENCHMARK_CRC32(braid, PREFIX(crc32_braid), 1);
```

Comment:
Do we want to benchmark crc32_braid along side crc32_c? It might be a good idea.

Reply:
done